charts/rancher-monitoring/v0.0.7000/values.yaml

```diff
@@ -396,3 +396,21 @@ operator-init:
     requests:
       cpu: 20m
       memory: 20Mi
+
+webhook-receiver:
+  enabled: false
```

Comment:
it is better to add default values to the `webhook-receiver/values.yaml` since it follows the chart design standard.